db/seeds/places.rb

```diff
@@ -8,12 +8,27 @@
   "places.csv"
 )
 
+duplicates_filename = File.join(
+  Rails.root,
+  "db",
+  "data",
+  "missing_photos.csv"
+)
+
+duplicates = CSV.parse(
```

Comment:
The name `duplicates` confuses me. Could you rename it to better reflect its purpose, e.g. `inactive_images`?